index.html

```diff
@@ -23,22 +23,147 @@ <h1>Hey There.i'm Said</h1>
+        </form>
+      </section>
+      <section class="work projectq">
+        <div class="linfoq">
+          <h2>Profesional Art Printing Data</h2>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="langq">
+            <li class="htq">html</li>
+            <li class="boq">bootstrap</li>
+            <li class="ruq">Ruby</li>
+          </ul>
+        </div>
+        <form>
+          <button type="button" class="butq">See Project</button>
+        </form>
+      </section>
+      <section class="work projectw">
+        <div class="linfow">
+          <h2>Profesional Art Printing Data</h2>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="langw">
+            <li class="htw">html</li>
+            <li class="bow">bootstrap</li>
+            <li class="ruw">Ruby</li>
+          </ul>
+        </div>
+        <form>
+          <button type="button" class="butw">See Project</button>
+        </form>
+      </section>
+      <section class="work projectX">
+        <div class="linfoe">
+          <h2>Profesional Art Printing Data</h2>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="leng">
+            <li class="hte">html</li>
+            <li class="strap">bootstrap</li>
+            <li class="rube">Ruby</li>
+          </ul>
+        </div>
+        <form>
+          <button type="button" class="bute">See Project</button>
+        </form>
+      </section>
+      <section class="work about">
+        <h2>About me</h2>
+        <p class="par">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+        <form>
+          <button type="button" class="abouut">Get My Resume</button>
+        </form>
+        <div class="vic"></div>
+        <section class="langue">
+          <div class="Eng">
+            <h3 class="title0">Language</h3>
+            <section class="map">
+              <ul>
+                <li>Javascript</li>
+                <li>Ruby</li>
+                <li>Html</li>
+                <li>Css</li>
+              </ul>
+            </section>
+            <h3 class="title1">Frameworks</h3>
+            <section class="map">
+              <ul>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+                <li>CapyBara</li>
+                <li>Selenium</li>
+              </ul>
+            </section>
+            <h3 class="title2">Skills</h3>
+            <section class="map">
+              <ul>
+                <li>Codekit</li>
```

Comment:
- [ ] You should horizontally center the contents of this section by the help of css FlexBox or Grid layout as in the Figma design.
#### Impl
![image](https://user-images.githubusercontent.com/44057863/170739229-bca4e7a9-26c7-4e94-a1e3-3b087598c965.png)
#### Design
![image](https://user-images.githubusercontent.com/44057863/170739309-e8b81b43-9fd9-49cd-8425-634993afd42e.png)

Reply:
done

Reply:
Kindly add the decorations on the group headers.
|Figma Design| Your Design|
|---|---|
|![image](https://user-images.githubusercontent.com/12239821/170780423-ef08cf2a-6a96-4d72-be63-eb9380cee7cb.png)|![image](https://user-images.githubusercontent.com/12239821/170780503-f4b5b118-8aae-4828-baac-335855ba8e9a.png)|

Reply:
done @chrystalme